src/ir/lowering.rs

```diff
@@ -356,7 +365,7 @@ fn lower_struct(
         let variant = VariantDef {
             def_id: ctx.gen.next_defid(),
             name: field.name.name.clone(),
-            ty: lower_type(&ctx, &field.r#type, module_id)?,
+            ty: lower_type(&ctx, &field.r#type, module_id, None)?, // todo: struct generics
```

Comment:
do we have an issue for this TODO? I think it's a good idea to also include the associated issue into the todo.

Something like:

```rust
// TODO(#180): implement generics in structs
```